.gitignore

```diff
@@ -2,30 +2,15 @@
 .vscode
 .DS_Store
 
-/beta
-/build
+# XCode project files
 Transmission.xcodeproj/*.xcworkspace
 Transmission.xcodeproj/xcuserdata
-/cli/transmissioncli
-/daemon/transmission-daemon
-/daemon/transmission-proxy
-/daemon/transmission-remote
-/gtk/defines.h
-/gtk/transmission-gtk
-/libtransmission/*.a
-/libtransmission/version.h
+
+# files Xcode creates
 /macosx/Info.plist
-/macosx/**/*~.nib
-node_modules/
-/po/*.mo
-/REVISION
```

Comment:
At least for pure Mac+Xcode build these are needed to be ignored:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	REVISION
	libtransmission/version.h
	third-party/suffixes_dafsa.h
```

I have just checked out your PR branch.

As I said before, some steps in Xcode build are generating files in tree.

Command line used:
```
xcodebuild -project Transmission.xcodeproj -scheme Transmission -configuration Debug -destination 'platform=macOS,arch=arm64' build
```

Reply:
Thanks for this feedback, this is exactly what I was looking for. I'll add these back.